tests/st/policy/test_profile.py

```diff
@@ -30,15 +30,16 @@
 class MultiHostMainline(TestBase):
     @parameterized.expand([
         #"tags",
-        "rules.tags",
+        ("rules.tags", False),
+        ("rules.tags", True),
```

Comment:
I don't think this should be part of the policy test (unless you think we need to be testing this with different sets of policy.  

We have a `test_ipip.py` in the `calicoctl` tests which I think could be moved over to the bgp tests (since it's more testing Bird) - and then maybe we could include this test in test_ipip.py.